cmd/user-agent/src/main.js

```diff
@@ -80,10 +80,11 @@ async function ariesStartupOpts() {
     }
 }
 
-let defaulTrustBlocStartupOpts = {
+let defaultTrustBlocStartupOpts = {
     assetsPath: '/trustbloc-agent/assets',
     blocDomain: 'testnet.trustbloc.local',
-    walletMediatorURL: ''
+    walletMediatorURL: '',
+    'log-level': 'debug'
```

Comment:
I noticed that the Aries agent uses a default log level of "debug" as the default... so I used "debug" here as well for consistency. Was the choice of "debug" for the aries agent intentional? Or should I change it to "info" (which means I can just remove this parameter since info is the default anyway)?

@sudeshrshetty @fqutishat @rolsonquadras